packages/manager/modules/dbaas-logs/src/logs/detail/roles/enable-iam/logs-iam-enable.controller.js

```diff
@@ -0,0 +1,61 @@
+export default class LogsIamEnableModalCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $stateParams,
+    $uibModalInstance,
+    $translate,
+    CucControllerHelper,
+    CucCloudMessage,
+    LogsHomeService,
+  ) {
+    this.$stateParams = $stateParams;
+    this.$q = $q;
+    this.CucControllerHelper = CucControllerHelper;
+    this.CucCloudMessage = CucCloudMessage;
+    this.LogsHomeService = LogsHomeService;
+    this.$uibModalInstance = $uibModalInstance;
+    this.$translate = $translate;
+    this.serviceName = $stateParams.serviceName;
+    this.initLoaders();
+  }
+
+  $onInit() {
```

Comment:
Make more sense to be defined in `$onInit` function
```suggestion
     }

  $onInit() {
   this.initLoaders();
```